src/Umbraco.Infrastructure/Mail/EmailSender.cs

```diff
@@ -28,26 +26,29 @@ public class EmailSender : IEmailSender
     private readonly ILogger<EmailSender> _logger;
     private readonly bool _notificationHandlerRegistered;
     private GlobalSettings _globalSettings;
-
+    private readonly IEmailSenderClient _emailSenderClient;
     public EmailSender(
         ILogger<EmailSender> logger,
         IOptionsMonitor<GlobalSettings> globalSettings,
-        IEventAggregator eventAggregator)
-        : this(logger, globalSettings, eventAggregator, null, null)
+        IEventAggregator eventAggregator,
+        IEmailSenderClient emailSenderClient)
+        : this(logger, globalSettings, eventAggregator, emailSenderClient,null, null)
```

Comment:
Besides the breaking changes it looks good now.

Please add the original constructor signatures and use `StaticServiceProvider.Instance.GetRequiredService<IEmailSenderClient>()` where needed

Reply:
Gotcha - it's back to normal now and using StaticServiceProvider instead

Reply:
@bergmania - no I'm sorry, I've actually had to do that in another pull request so I should have guessed it.

Here is another attempt, though speaking of constructors, I'm a bit perplexed by the shorthanded constructor 

```csharp
public EmailSender(
    ILogger<EmailSender> logger,
    IOptionsMonitor<GlobalSettings> globalSettings,
    IEventAggregator eventAggregator)
    : this(logger, globalSettings, eventAggregator,null, null)
{
}
```

as it has no references to it in the entire code base? If it's there for backwards compatibility, I suppose it should also be obsolete ? That's what I've done anyway. Hope I'm right.